job-descriptions/software-engineer-security.md

```diff
@@ -2,20 +2,35 @@
 
 # Software Engineer - Security
 
-We are looking for a software engineer to own the security of our product. This includes the security of sourcegraph.com as well as making it easy for our customers to secure their own private Sourcegraph instances that they deploy on-premise. You will be responsible for proactively understanding, documenting, and improving the security of our product, as well as promptly responding to and addressing security issues reported to us by third parties. As our security needs grow over time, there is a potential for you to grow into a security team leadership role.
+We are looking for an expert to own security at Sourcegraph. This includes the security of Sourcegraph.com as well as making it easy for our customers to secure their own private Sourcegraph instances that they deploy on-premise. You will have the opportunity to build the best security culture and organization in the world from the ground up.
```

Comment:
```suggestion
We are looking for an expert to own security at Sourcegraph. This includes the security of Sourcegraph.com as well as making it easy for our customers to secure their own private Sourcegraph instances that they deploy on-premise. You will have the opportunity to establish and build a world-class security and organizational practices from the ground up.
```

Current wording sounds a bit far out there; we aren't a security firm so how could we possibly be _the_ best? :)

Reply:
Hah, my first draft used the words “world class” and then I made it more aggressive because why not, this is supposed to be inspirational :)

I’ll think about it.

Reply:
```suggestion
We are looking for an expert to own security at Sourcegraph. This includes the security of Sourcegraph.com as well as making it easy for our customers to secure their own private Sourcegraph instances that they deploy on-premise. You will have the opportunity to build a world-class security culture and team from the ground up.
```